app/views/admin/reports/show.html.haml

```diff
@@ -1,7 +1,3 @@
-- content_for :header_tags do
-  = javascript_pack_tag 'admin', async: true, crossorigin: 'anonymous'
-  = javascript_pack_tag 'public', async: true, crossorigin: 'anonymous'
```

Comment:
The `public` removal here is also intentional - that's already included by the admin layout before this PR.